src/test/java/io/appium/java_client/touch/TouchOptionsTests.java

```diff
@@ -71,7 +72,7 @@ public void longPressOptionsShouldBuildProperly() {
     @Test
     public void tapOptionsShouldBuildProperly() {
         final Map<String, Object> actualOpts = tapOptions()
-                .withPosition(point(0, 0))
+                .withPosition(point(new Point(0, 0)))
```

Comment:
so now both would be possible `point(0, 0`) and `point(new Point(0, 0))` ?

Reply:
Yes both are possible now.